tests/unittests/clustering/inputs.py

```diff
@@ -23,10 +24,15 @@
 
 
 # extrinsic input for clustering metrics that requires predicted clustering labels and target clustering labels
-ExtrinsicInput = namedtuple("ExtrinsicInput", ["preds", "target"])
+class _ExtrinsicInput(NamedTuple):
```

Comment:
seems to be the same as `_Input` in other files so why different name?

Reply:
This was to differentiate between input types for the extrinsic and intrinsic clustering methods. Other metrics only have one type of input (extrinsic) so there is no need to name the variable.

Should we change `_ExtrinsicInput` to `_Input` and keep `_IntrinsicInput` unchanged?

Reply:
yes that sound slike a good trade